test/src/tests.js

```diff
@@ -204,10 +206,12 @@ describe('Heap Forwarder', function () {
     it('should initialize Heap', function (done) {
         mParticle.forwarder.init({
             appId: 'test-app-id',
+            forwardWebRequestsServerSide: false,
         });
 
         window.heap.should.be.ok;
-        window.heap.appid.should.equal('test-app-id');
+        window.heap.track.should.be.ok;
+        window.heap.envId.should.equal('test-app-id');
```

Comment:
I would recommend adding a comment here for posterity so that if we look at this again in six months (or longer) we aren't forgetting this specific nuance of the Heap init.

```suggestion
        // The Heap SDK initializes with a stubbed object that contains 
        //  `envId: <app-id>` but will be replaced by `appId: <app-id>` once 
        //  Heap fully initializes. For the purposes of our test framework,
        // we test to make sure the stubbed methods are initialized. 
        window.heap.envId.should.equal('test-app-id');
```